### DISCUSSION TITLE
BE_SQLITE_ERROR

### DISCUSSION THREAD

**Discussion by nn15296** [iTwin/itwinjs-core#1719] [Q&A] [answered]
Any ideas where this error comes from? It occurs when I write extracted imodel information to a third party database. However, it has only occurred this once having processed many other imodels successfully. Is it potentially as a result of it processing 41697 imodel elements? Does the briefcase.withPreparedStatement() method timeout perhaps?

Thanks

![image](https://user-images.githubusercontent.com/33571096/123450151-545b8880-d5d4-11eb-92bb-bc657cef343d.png)

**Comment by pmconne** [comment, marked as answer]
Our ability to assist you will be extremely limited if you don't show your code.

**Reply by pmconne** [reply]
Closing in absence of follow-up information.
